common/v2/database/generateDefaultValues.ts

```diff
@@ -15,16 +15,22 @@ import {
   Fiat,
   ContractLegacy,
   AssetLegacy,
-  LSKeys
+  LSKeys,
+  NodeOptions
 } from 'v2/types';
 
+import { NODES_CONFIG, NETWORKS_CONFIG, NetworkConfig } from './data';
 import { SeedData, StoreAction } from './types';
 import { toArray, toObject, add } from './helpers';
+
 /* Transducers */
 const addNetworks = add(LSKeys.NETWORKS)((networks: SeedData) => {
   const formatNetwork = (n: NetworkLegacy): Network => {
     const baseAssetUuid = generateAssetUUID(n.chainId);
-    const nodes = NODES_CONFIG[n.id] || NODES_CONFIG[n.name as NetworkId] || [];
+    const nodes: NodeOptions[] = NODES_CONFIG[n.id] || [];
+    if (n.nodes) {
+      nodes.push(...n.nodes); // add custom nodes
+    }
```

Comment:
Couldn't this just be: 
```
const nodes: NodeOptions[] = [...(NODES_CONFIG[n.id] || []), ...n.nodes];
```
?

Reply:
This will fail since `n.nodes` can be undefined, but I slightly changed it to:
```
const nodes: NodeOptions[] = [...(NODES_CONFIG[n.id] || []), ...(n.nodes || [])];
```